src/main/java/uk/gov/ons/census/casesvc/logging/EventLogger.java

```diff
@@ -69,7 +69,7 @@ public void logRefusalEvent(
     if (StringUtils.isEmpty(event.getTransactionId())) {
       loggedEvent.setEventTransactionId(UUID.randomUUID());
     } else {
-      loggedEvent.setEventTransactionId(UUID.fromString(event.getTransactionId()));
+      loggedEvent.setEventTransactionId(event.getTransactionId());
```

Comment:
hurrah for duplication

Reply:
@LukeLoze1 I'm not sure why these `log*Type*Event` methods exist, non of them appear to do anything specific to the event type in their name and they could be replaced with one or two versions of the general `logEvent` method (like I used in this PR to log the UAC created events). I thought that refactor was out of the scope of this change though.